v1.1: add `csvDelimiter` to add/update/replace document endpoints (#2223)
meilisearch/documentation: reference/errors/error_codes.md

The [Content-Type header](/reference/api/overview.md#content-type) is not supported by Meilisearch. Currently, Meilisearch only supports JSON, CSV, and NDJSON.

## `invalid_document_csv_delimiter`

> The value given to [`csvDelimiter`](/reference/api/documents.md#add-or-replace-documents) is either not a string, or is not a single ASCII character.

Reviewer: ```suggestion
The [`csvDelimiter`](/reference/api/documents.md#add-or-replace-documents) parameter is invalid. It should either be a string or [a single ASCII character](https://www.rfc-editor.org/rfc/rfc20).
```
Reworded to it to match the other errors
